package.json

```diff
@@ -46,6 +46,7 @@
   "devDependencies": {
     "@babel/eslint-parser": "~7.24.1",
     "@babel/plugin-syntax-import-assertions": "~7.24.1",
+    "@ddbeck/mdn-content-inventory": "^0.1.0-20240618.c16ab79591",
```

Comment:
If you decide to go forward with this, one thing you might want to watch out for (or something I could tend to upstream) is versioning.

For my use of this package, I need to be able to get an inventory by date (e.g., I want see the state of the repo on YYYY-MM-DD), so I've done a bit of a hack, with the semver pre-release notation (the `-` and everything after).

Unfortunately, this means that npm and Dependabot's semver handling won't automatically pick up these "pre-releases." Either you'll need to have a custom workflow to periodically upgrade this dependency or I'll need to adopt some other versioning scheme (e.g., `0.1.20240618` or similar).

Reply:
Hm, I wonder if dependabot updates would be bes.  We could then run "npm run fix" on that branch to fix issues because we will likely be observing newly written MDN pages upon updating this.

Reply:
Or a custom workflow, as you say, which would pick up your versioning scheme and runs `npm run fix`

Reply:
OK, I'll try the `0.1.YYYYMMDD` versioning scheme and see how that goes.

Reply:
The package is now auto-publishing under the `x.y.YYYYMMDD` scheme ([example](https://www.npmjs.com/package/@ddbeck/mdn-content-inventory/v/0.1.20240620)), if you want to `npm install --save-dev @ddbeck/mdn-content-inventory@latest` now.